windows.md

```diff
@@ -448,7 +448,7 @@ With those compatibility things out of the way, you're ready to start the system
 
     ```bash
     choco install androidstudio
-    echo "export PATH=$HOME/AppData/Local/Android/Sdk/platform-tools:\$PATH" >> ~/.bash_profile
+    echo "export PATH=~/AppData/Local/Android/Sdk/platform-tools:\$PATH" >> ~/.bash_profile
```

Comment:
this one is from the React Native / Expo setup guide, right? I think they use `$HOME` there - and we also use `$HOME` in all the other guides

Reply:
Reverted the changes
[ac7e243](https://github.com/upleveled/system-setup/pull/94/commits/ac7e2439382a7da638c84a653be121790bf81c81)